src/app/menu/menu.component.ts

```diff
@@ -10,9 +10,13 @@ import { LanguageService } from '../services/language.service';
 })
 export class MenuComponent implements OnInit {
 
```

Comment:
Can all the class name strings be made constants in this file?

Reply:
Done!